### PR TITLE
Remove duplicate setup instructions from android custom build page

### LINKED ISSUE #3469: Duplicated Export Documentation
**Your Godot version:** V3.2.1

**Issue description:**
Currently there are two documents describing how to export for Android. See links below.
They differ regarding the installation procedure for several aspects like Android SDK and JDK.

I suggest to streamline these two documents, so that they do not differ regarding dependencies installation.

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/latest/getting_started/workflow/export/exporting_for_android.html
https://docs.godotengine.org/en/latest/getting_started/workflow/export/android_custom_build.html

### LINKED ISSUE #3470: Update Android SDK installation procedure for android custom build documentation
**Your Godot version:** v3.2.1

**Issue description:**

Because of changes in the android-sdk, the custom android build documentation currently no longer works as described.
Problem is the directory structure

    android-sdk/
    android-sdk/tools/
    android-sdk/tools/allthefiles

According to the answer https://stackoverflow.com/a/61176718 that worked for me, the directory structure needs adjustments.

**URL to the documentation page:**
https://docs.godotengine.org/en/latest/getting_started/workflow/export/android_custom_build.html

### LINKED ISSUE #3689: Custom Builds for Android doc has wrong instructions
**Your Godot version:**
3.2.1.stable.official
**Issue description:**
I had to install a custom export template to include ads in my game. I already had the _Android/Sdk_ folder. I have to rename _tools_ to _latest_, create a folder: _cmdline-tools_ and place _latest_ into it. In the next step I have to "Navigate to the tools/bin" but wait, I don't have this folder anymore! I had to rename it to latest, so the commands didn't work, I had plenty of errors, so could someone explain to me please more accurate?
**URL to the documentation page:**
https://docs.godotengine.org/en/latest/getting_started/workflow/export/android_custom_build.html#download-the-command-line-tools
https://docs.godotengine.org/en/latest/getting_started/workflow/export/android_custom_build.html#accepting-the-licenses

### PR DESCRIPTION
**Pull request by skyace65**
The exporting for android page already covers the information that I've removed from the custom builds page, and the custom builds page now tells you what you need to do from the setup page before making a custom build.

This makes the documentation significantly easier to maintain as we're not working with two different sets of setup instructions. I know we've had issues in the past with the two pages not giving the same information. If this is good to merge I'll make another PR for 3.3 branch so it can get added to stable. Though I want to close some other android issues with PRs before stable gets updated.

Closes #3469, closes #3470, closes #3689

### CODE REVIEW

File: tutorials/export/android_custom_build.rst [hunk 1]
```diff
@@ -13,10 +13,9 @@ There are some reasons why you may want to do this:
 * Modify the project before it's built.
 * Add external SDKs that build with your project.
 
-Configuring the custom build is a more or less straightforward process,
-but it may take a while to get used to how the Android SDK works.
-
-Instructions will be provided as detailed as possible to do this process.
+Configuring the custom build is a more or less straightforward process. But first
+you need to do the steps in :ref:`exporting for android<doc_exporting_for_android>`
+up to "Setting it up in Godot." After doing that follow the steps below.
```
Comment (Calinou) [inline tutorials/export/android_custom_build.rst:18]:
```suggestion
Configuring the custom build is a fairly straightforward process. But first,
you need to follow the steps in :ref:`exporting for android<doc_exporting_for_android>`
up to **Setting it up in Godot**. After doing that, follow the steps below.
```

**Review by Calinou — CHANGES_REQUESTED** [review]

**Comment by skyace65** [comment]
Fixed

**Review by m4gr3d — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.3 (not 3.2 as I think the instructions do differ there).

**Comment by skyace65** [comment]
@akien-mga can you cherry pick this one since the exporting pages are in a different location? If you need me to open a second PR for 3.3 I can do that.

**Comment by akien-mga** [comment]
As I wrote above I cherry-picked it already.

**Pull request merged**